onboarding/target/task_descriptions.js

```diff
@@ -1,100 +1,63 @@
-(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
 hasMouse = !('ontouchstart' in window);
 
 taskDescriptions = {
   descriptions: [hasMouse ? {
-    key: 'Drag', label: 'Drag this data file into CODAP', url: './resources/DragCSV.mp4',
+    key: 'Drag', label: tr("~onboarding1.drag.task"), url: './resources/'+tr("~onboarding1.drag.movie.filename"),
     operation: 'dataContextCountChanged',
     feedback: React.createElement(
       'div',
       null,
       React.createElement(
         'p',
         null,
-        'You\'ve got data! It appears in a ',
-        React.createElement(
-          'em',
-          null,
-          'case table'
-        ),
-        '.'
+        tr("~onboarding1.drag.success.message1")
       ),
       React.createElement(
         'p',
         null,
-        'Each row in the table represents a ',
-        React.createElement(
-          'em',
-          null,
-          'case'
-        ),
-        ' and each column represents an ',
-        React.createElement(
-          'em',
-          null,
-          'attribute'
-        ),
-        '.'
+        tr("~onboarding1.drag.success.message2")
       ),
       React.createElement(
         'p',
         null,
-        'This data set contains data about mammals. Each case represents a different mammal. The attributes provide information about lifespan, height, and so on.'
+        tr("~onboarding1.drag.success.message3")
       )
     )
   } : {
-    key: 'MakeTable', label: 'Make a table showing Mammals data', url: './resources/MakeTable.mp4',
+    key: 'MakeTable', label: tr("~onboarding1.make.table.task"), url: './resources/' + tr("~onboarding1.make.table.movie.filename"),
```

Comment:
It is likely the someone on the Concord side of things will fill in these values in POEditor, so that we can group translated videos in different subfolders. It's not really about having a translated name of a file, but that could be helpful too if they send us filenames that have translated names. For example above the "MakeTable.mp4" file for Spanish is named the same way, but placed in a sub-folder, such that it is found at /es/MakeTable.mp4. Putting this in the translation project in POEditor makes it so that we don't have to hardcode this in the plugin code.